msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -19,8 +24,9 @@ Proactive messages can be useful in a variety of scenarios. These are some examp
 ### Basic guidelines
```

Comment:
I don't think this entire section applies to Teams. Proactive messaging in teams is typically a welcome message or notification in a 1:1 conversation, or a notification into a channel or group chat. Very rarely would a proactive message occur in the midst of an on-going conversation, or need to break into a dialog. Please remove this and replace with the opening from the existing proactive message article.

Reply:
Made requested changes.